ddtrace/span.py

```diff
@@ -198,7 +208,7 @@ def set_tag(self, key, value=None):
                 # DEV: `set_metric` will try to cast to `float()` for us
                 self.set_metric(key, value)
             except (TypeError, ValueError):
-                log.debug('error setting numeric metric %s:%s', key, value)
+                log.warning("error setting numeric metric %s:%s", key, value)
```

Comment:
I changed these to be warnings... I think it's warranted to have an elevated severity for erroneous behaviour.